core/src/templates/components/date-stacked/date-stacked.twig

```diff
@@ -0,0 +1,24 @@
+{#
+/**
+ * @file
+ * Date Stacked Component.
+ *
+ * Abbreviated month stacked on top of numerical day of the month.
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * - month_of_year: The abbreviated month of the year.
+ * - day_of_month: The day of the month in digit form.
+ */
+#}
+<div{{ attributes }} class="su-date-stacked {{ modifier_class }}">
+  {% if month_of_year is not empty %}
+    <div class="su-date-month">{{ month_of_year }}</div>
+  {% endif %}
+
+  {% if day_of_month is not empty %}
+    <div class="su-date-day">{{ day_of_month }}</div>
+  {% endif %}
+
+</div>
```

Comment:
Do we want to change the container from a `<div>` to a `<time datetime="...">`? (See `<time>` on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).) In that case we might also want a year. The year wouldn't (necessarily) render in the browser, but bots could recognize the actual date.

Reply:
I'll explore this more.

Reply:
That looks handy. It appears it doesn't have IE support and we should still support IE 11 so my question is what would it look like in IE11? My guess is that it isn't supported but probably looks 100% fine and should be usable. I have been tricked before however.

Reply:
[caniuse](https://caniuse.com/#feat=html5semantic) > **Known issues** says:

> While the `time` and `data` elements can be used and work fine in all browsers, currently only Firefox and Edge 14+ recognize them officially as `HTMLTimeElement` and `HTMLDataElement` objects.

In fact, we use it on news.stanford.edu. Go to any story on that site and inspect the date the story was published - it's a `<time>` tag. We should of course test it in IE, but it should be fine.

I'm not at all clear on how much benefit it offers, but I'd still lean towards using `<time datetime="...">`.

Reply:
I don't think it's meaningful to just have `{{ month_of_year}}` or `{{ day_of_motnh }}` in the `datetime` attribute - see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Valid_datetime_Values). You could end up with `datetime="9"`. What would that mean?

I think we should do our best to  provide the most specific datetime possible, even if that mean requiring data that doesn't render visually. I think we should at least have `datetime="{{ year }}-{{ month_of_year}}-{{ day_of_month }}"`, which requires a year, even though no year renders visually. (Of course we'd need logic to build the string based on what data we actually have - we don't want dangling dashes.) We might also add some logic to include the time, if provided.

Reply:
I am for having the time tag in there with a useful datetime but I am unsure how to make anything required that doesn't visually impact the component. Perhaps we split the difference and provide the input variables for the proper datetime with month-day-year time and timezone and use Twig's date functions to default to the server environment: https://twig.symfony.com/doc/2.x/filters/date.html

Reply:
@sherakama  I'm looking at this:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Seems like a yearless date string in the form of 12-30 for example is valid. Could we do something like <time datetime="{{ month }}-{{ day }}><span>{{ day }}</span><span>{{ month }}</span></time>?

Reply:
How about now?

Reply:
Ideally it would be nice to have just one `<time>` tag. Also, `datetime` with just the month alone might not be valid without a year attached? (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time). I'm fine with the current version though, except maybe only emit one `datetime` attribute so it doesn't repeat, if we're using 2 `<time>` tags? Other than that GTG.

Reply:
🤔 hmmm. Perhaps we go back to the `<span>` tag for safety. I know it would be more semantic/accessible if the field used `<time>` but if we are to allow for a mix and match approach, I wouldn't want to have tag switching on the component as that could leave to confusing UI or behaviors if they alternated in a list or something.

Reply:
That's true....if you're anticipating cases that either the day or month might be missing, then `<span>` is a safer choice, since a valid time tag would require both a month and a day 🤔

Reply:
Ok, back to plain ol' span until we come up with a better solution.